### PR TITLE
[IMP] settings: simplify the developer mode page

### PR DESCRIPTION
**Pull request by xpl-odoo**
task-3849915

### CODE REVIEW

**Review by auva-odoo — APPROVED** [review]
@xpl-odoo Great job on reviewing this page (and getting rid of all of those ugly red boxes; defintely a big win in my book :ok_hand:). I'll go ahead and approve this already but I added 2 suggestions (see my comments), and I also think adding a warning somewhere at the beginning is a good idea. Hope this helps! :slightly_smiling_face:

File: content/applications/general/developer_mode.rst [hunk 1]
```diff
@@ -1,100 +1,55 @@
...
-========================
+.. image:: developer_mode/settings.png
+   :alt: Activating the developer mode in the Settings app
 
-The debug mode can also be activated by adding an extra query string to the database's URL. In the
-URL, add `?debug=1` or `?debug=true` after `/web`. To deactivate the debug mode, change the
-value to `?debug=0` instead.
+.. tip::
+   Open the **command palette** by pressing `Ctrl + K` or `Cmd ⌘ + K`, then type `debug` to
+   activate the developer mode with assets or deactivate it.
```
[the remark this answers is not in the document]
Comment (xpl-odoo) [reply content/applications/general/developer_mode.rst:28]: I will move the tip, but I think I will leave the _deactivate_ part at the end, as you can activate the developer mode _with assets_ (only that one) or deactivate the developer mode (with assets, tests assets, or without either) if it was activated previously. Putting activate/deactivate together makes it seem you can only deactivate the developer mode with assets :)

**Comment by xpl-odoo** [comment]
@robodoo r+

**Comment by xpl-odoo** [comment]
@robodoo retry

**Pull request closed without merging**